docs/environment.md

```diff
@@ -0,0 +1,53 @@
+# Setting up the common environment
+
+Log in to the LSF cluster, where all data processing must take place. Switch to a common EVA production user instead of your personal account. Then adjust and execute the commands below. They will set up the environment, fetch and build the code.
+
+Notes:
+* The first five variables are installation-specific and are blanked in this repository. You can get the values for the EVA use case from the [private repository](https://github.com/EBIvariation/configuration/blob/master/open-targets-configuration.md).
+* By modifying the `*REMOTE` and `*BRANCH` variables, you can run arbitrary versions of both the main and the VEP pipeline. This is highly useful for development and debugging. By default it fetches master branches of both repositories.
+
+```bash
+# This variable should point to the directory where the clone of this repository is located on the cluster
+export CODE_ROOT=
+
+# Location of Python installation which you configured using build instructions
+export PYTHON_INSTALL_PATH=
+
+# Location of bcftools installation path
+export BCFTOOLS_INSTALL_PATH=
+
+# The directory where subdirectories for each batch will be created
+export BATCH_ROOT_BASE=
+
+# Base path of FTP directory on the cluster
+export FTP_PATH_BASE=
+
+# Base bsub command line for all commands.
+export BSUB_CMDLINE="bsub"
+
+# Setting up Python paths
+export PATH=${PYTHON_INSTALL_PATH}:${PYTHON_INSTALL_PATH}/bin:${BCFTOOLS_INSTALL_PATH}:$PATH
+export PYTHONPATH=${PYTHON_INSTALL_PATH}
+
+# External service paths
+CLINVAR_PATH_BASE="ftp://ftp.ncbi.nlm.nih.gov/pub/clinvar"
+
+export MAIN_REMOTE=origin
+export MAIN_BRANCH=master
+export VEP_REMOTE=origin
+export VEP_BRANCH=master
+
+cd ${CODE_ROOT}
+git fetch ${MAIN_REMOTE}
+git checkout ${MAIN_BRANCH}
+git reset --hard ${MAIN_REMOTE}/${MAIN_BRANCH}
+
+git submodule update --init --recursive
+cd vep-mapping-pipeline
+git fetch ${VEP_REMOTE}
+git checkout ${VEP_BRANCH}
+git reset --hard ${VEP_REMOTE}/${VEP_BRANCH}
+cd ..
+
+python3 setup.py install
```

Comment:
Also would it be better to use a virtual environment rather than the base install ?

Reply:
> This will install dependencies from `eva-cttv-pipeline` but will not install the dependencies from `vep-mapping-pipeline`.

Actually no, this would install _none_ of the requirements, because `requirements.txt` is not automatically caught by `setup.py` (for neither the root package, nor subpackages). So thank you for finding this, it's a pretty serious issue. I guess the reason everything has been working so far is that those requirements were installed many months ago when first setting up the package, and they just have been there ever since.

Fixed by explicitly specifying install instructions: https://github.com/EBIvariation/eva-cttv-pipeline/pull/99/commits/aed78167b4036a0f033b19ce369ff06dc25587c7

Reply:
Regarding your second question: a virtual environment would probably be a bit better approach. However, I really don't want to bloat the scope of this ticket even more. Do you mind if I do this as a separate small ticket at some point in the future?

Reply:
> Regarding your second question: a virtual environment would probably be a bit better approach. However, I really don't want to bloat the scope of this ticket even more. Do you mind if I do this as a separate small ticket at some point in the future?
You're right. Several of my comments have nothing to do with your changes.
Feel free to just push the ones that seems irrelevant to other tickets

Reply:
Literally all of your comments have been very helpful :)

I have moved this into https://www.ebi.ac.uk/panda/jira/browse/EVA-1982, along with unifying the clinical significance levels in one place and removing some unused modules which are still present.